includes/class-indieauth-authorize.php

```diff
@@ -210,23 +210,23 @@ public function get_provided_token() {
 		return null;
 	}
 			/**
-	 * Extracts the token from the given authorization header.
-	 *
-	 * @param string $header Authorization header.
-	 *
-	 * @return string|null Token on success, null on failure.
-	 */
+			 * Extracts the token from the given authorization header.
+			 *
+			 * @param string $header Authorization header.
+			 *
+			 * @return string|null Token on success, null on failure.
+			 */
 	public function get_token_from_bearer_header( $header ) {
 		if ( is_string( $header ) && preg_match( '/Bearer ([\x20-\x7E]+)/', trim( $header ), $matches ) ) {
 			return $matches[1];
 		}
 		return null;
 	}
 			/**
-	 * Extracts the token from the current request.
-	 *
-	 * @return string|null Token on success, null on failure.
-	 */
+			 * Extracts the token from the current request.
```

Comment:
Indents: Please correct the alignment of the code block

Reply:
I don't know why phpcs keeps pulling things out of alignment. It is supposed to identify that. And this PR is all about updating it, so I will be making the rounds of the other plugins